test/fixtures/security/contracts-with-free-functions.js

```diff
@@ -0,0 +1,176 @@
+// all contracts are made with two func-visibility errors
+
+module.exports = [
```

Comment:
I would prefer these tests to be simpler and to do one thing each. For example, a simple test with just a free function that doesn't throw errors; another test that has a free function and a contract function without visibility; etc. I realize that this means having them in different files and being more explicit in the test file (importing each fixture in a different test because it produces a different number of warnings), but it's worth it.

Reply:
done